src/main/java/com/filestack/ImageTags.java

```diff
@@ -0,0 +1,25 @@
+package com.filestack;
+
+import java.util.Map;
+
+/**
+ * Tags returned from Google Vision API.
+ *
+ * @see <a href="https://www.filestack.com/docs/tagging"></a>
+ */
+public class ImageTags {
+  Tags tags;
+
+  class Tags {
+    Map<String, Integer> auto;
+    Map<String, Integer> user;
+  }
+
+  public Map<String, Integer> getAuto() {
+    return tags.auto;
+  }
+
+  public Map<String, Integer> getUser() {
```

Comment:
I wonder if we should have this here, as we don't have image tags yet? Maybe there is no reason not to, but it currently will never return anything, so it could be confusing. I wonder if we should remove the user parameter from all the SDK tags responses until it's an actual feature?

Reply:
Agree. Let's remove it for now.

Reply:
I left the ImageTags object the same but moved it into the responses module and updated the tags method. So now the public interface looks like this:

```java
Map<String, Integer> tags = filelink.imageTags();
```

The user only sees a single map of tags now, not "auto" or "user".